omod/src/main/resources/config.xml

```diff
@@ -265,6 +265,14 @@
         </description>
     </globalProperty>
 
+    <globalProperty>
+        <property>conditionList.class</property>
```

Comment:
Did you take a look at the naming convention for the rest of the global properties in this file?

Reply:
i will update it

Reply:
Take a look at my comment again.